cognite/client/data_classes/simulators/simulators.py

```diff
@@ -0,0 +1,1619 @@
+    This object ensures that each execution of a routine is documented and traceable.
+    Each run has an associated simulation data resource, which stores the inputs and outputs of a
+    simulation run, capturing the values set into and read from the simulator model to ensure
+    the traceability and integrity of the simulation data.
+
+    Simulation runs provide a historical record of the simulations performed, allowing users to analyze
+    and compare different runs, track changes over time, and make informed decisions based on the simulation results.
+
+    This is the read/response format of a simulation run.
+
+    Args:
+        simulator_external_id (str | None): External id of the associated simulator
+        simulator_integration_external_id (str | None): External id of the associated simulator integration
+        model_external_id (str | None): External id of the associated simulator model
+        model_revision_external_id (str | None): External id of the associated simulator model revision
+        routine_external_id (str | None): External id of the associated simulator routine
+        routine_revision_external_id (str | None): External id of the associated simulator routine revision
+        run_time (int | None): Run time in milliseconds. Reference timestamp used for data pre-processing and data sampling.
+        simulation_time (int | None): Simulation time in milliseconds. Timestamp when the input data was sampled. Used for indexing input and output time series.
+        status (str | None): The status of the simulation run
+        status_message (str | None): The status message of the simulation run
+        data_set_id (int | None): The id of the dataset associated with the simulation run
+        run_type (str | None): The type of the simulation run
+        user_id (str | None): The id of the user who executed the simulation run
+        log_id (int | None): The id of the log associated with the simulation run
+        id (int | None): A unique id of a simulation run
+        created_time (int | None): The number of milliseconds since epoch
+        last_updated_time (int | None): The number of milliseconds since epoch
+
+    """
+
+    def __init__(
+        self,
+        simulator_external_id: str | None = None,
+        simulator_integration_external_id: str | None = None,
+        model_external_id: str | None = None,
+        model_revision_external_id: str | None = None,
+        routine_external_id: str | None = None,
+        routine_revision_external_id: str | None = None,
+        run_time: int | None = None,
+        simulation_time: int | None = None,
+        status: str | None = None,
+        status_message: str | None = None,
+        data_set_id: int | None = None,
+        run_type: str | None = None,
+        user_id: str | None = None,
+        log_id: int | None = None,
+        id: int | None = None,
+        created_time: int | None = None,
+        last_updated_time: int | None = None,
```

Comment:
I assume all these props don't have to be optional anymore, since we made a special `SimulationRunCreate` type, no?